### PR TITLE
pkg/logflags: Display warning when unknown component for --log-output

### PR DESCRIPTION
**Pull request by chainhelen**

### CODE REVIEW

File: pkg/logflags/logflags.go [hunk 1]
```diff
@@ -177,8 +180,11 @@ func Setup(logFlag bool, logstr string, logDest string) error {
 			fnCall = true
 		case "minidump":
 			minidump = true
-			// If adding another value, do make sure to
-			// update "Help about logging flags" in commands.go.
+		default:
+			if !unKnownHasPrint { // Just print first unknown component.
```
Comment (derekparker) [inline pkg/logflags/logflags.go]: I don't think we need to have this restriction, I don't mind printing a warning for each unknown log item.

File: pkg/logflags/logflags.go [hunk 2]
```diff
@@ -177,8 +180,11 @@ func Setup(logFlag bool, logstr string, logDest string) error {
...
+				fmt.Fprintf(os.Stderr, "Warning: unknown %q for '--log-output', run 'dlv help log' for usage.\n", logcmd)
```
Comment (derekparker) [inline pkg/logflags/logflags.go]: nit: `Warning: unknown log output value %s for ...`

File: pkg/logflags/logflags.go (hunk 1 above)
Comment (chainhelen) [reply pkg/logflags/logflags.go]: I have try, it looks ok. I will change and rebase.

File: pkg/logflags/logflags.go (hunk 2 above)
Comment (chainhelen) [reply pkg/logflags/logflags.go]: I remove `for '--log-output'` that the same meaning as `log output value` , avoid too long one line.

**Comment by chainhelen** [comment]
I will change and rebase.

**Pull request merged**